web/src/i18n/translations/en.yml

```diff
@@ -111,8 +111,14 @@ New Integration: ''
 Create: ''
 Integration Name: ''
 Please input the title of the integration!: ''
-Integration Token: ''
 Create new integration: ''
+Are you sure remove this intergration?: ''
+Permanently remove your Intergration and all of its contents from the Re:Earth CMS.: ''
+Once intergration was removed, it will disappear from all workspace.: ''
+Remove Intergration: ''
+Permanently remove your Intergration and all of its contents from the Re:Earth CMS. This action is not reversible – please continue with caution.: ''
```

Comment:
```suggestion
Once the integration is removed, it will disappear from all workspaces.: ''
Remove Integration: ''
Permanently remove your Integration and all of its contents from the Re:Earth CMS. This action is not reversible – please continue with caution.: ''
```